src/System.Net.Http.Rtc/pkg/System.Net.Http.Rtc.pkgproj

```diff
@@ -1,13 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think we should set up a case where we intentionally hinder our ability to produce an active shipping binary on one of our supported platforms so I'd lean towards not harvesting it.